spec.bs

```diff
@@ -2,18 +2,17 @@
 Title: Private State Token API
 H1: Private State Token API
 Shortname: private-state-token-api
-Level: 1
```

Comment:
CI says you need this? 🤷  I think `None` is correct here.

Reply:
Cool, thanks